### DISCUSSION TITLE
「发现开源三部曲」之一：《开源之迷》 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#32] [Announcements]
# 「发现开源三部曲」之一：《开源之迷》 - 开源之道

https://opensourceway.community/posts/book-of-open-source/the-fascinating-of-open-source/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣) 
应该改名为 开源之踪 作为三部曲开篇一本, 
基本作到了照顾所有类型读者的目标, 将开源相关的所有侧面都罗列出了靠谱资源, 
当然, 还是有问题: 

- 0: 作为彩印, 其中太多关键图片, 太模糊 
- 1: 竟然没有参考资料的索引, 如果逐一给出至少还能排42页, 多涨点价钱哪 
- 2: 作者留下了, 私人沟通邮箱, 带数字的, 一看就是互联网跟随者, 要是 gmail 发布之初受邀使用的, 一定能抢到没有数字的邮箱, 当然, 最好使用 开源之道 同名所有邮箱名哪... 
- 3: ... 

反正, 有了这一本, 至少启动了`中国开源学` 
当然, 作者是反感 中国XXX 的定义的,
不过, 毕竟是中国原创...


PS:

- 索引应该是出版商不同意
- 已经追加到当前网站中了:  [《开源之迷》参考材料汇总 \- 开源之道](https://opensourceway.community/posts/book-of-open-source/the-open-source-trilogy/the-fascinating-of-open-source-references/)
